src/librustc_metadata/decoder.rs

```diff
@@ -1193,6 +1207,10 @@ impl<'a, 'tcx> CrateMetadata {
 }
 
 fn are_equal_modulo_startpos(fm1: &syntax_pos::FileMap, fm2: &syntax_pos::FileMap) -> bool {
```

Comment:
(maybe we should move this to a method on `FileMap`, the idea being that future modifications to `FileMap`, such as adding new fields,  would know to update the method accordingly. But that can wait for a future PR; I prefer to keep this one minimal.)